core/src/main/java/discord4j/core/object/entity/Guild.java

```diff
@@ -1635,6 +1635,26 @@ public Mono<Void> unban(final Snowflake userId, @Nullable final String reason) {
                 .removeGuildBan(getId().asLong(), userId.asLong(), reason);
     }
 
+    /**
+     * Request a Bulk Ban to a specific list of users.
+     * <br>
+     * <b>Things considered error for this request</b>
+     * <ul>
+     *   <li>The list of users is over 200</li>
+     *   <li>None of the list of users can be banned</li>
+     * </ul>
+     *
+     * @param spec an immutable object that specifies how to bulk ban in the guild
+     * @return A {@link Mono} where, upon successful completion, emits an {@link BulkBan} with the results.
+     * If an error is received, it is emitted through the {@code Mono}.
+     */
+    public Mono<BulkBan> bulkBan(BulkBanSpec spec) {
+        Objects.requireNonNull(spec);
+        return gateway.getRestClient().getGuildService()
+            .bulkGuildBan(getId().asLong(), spec.asRequest(), spec.reason())
+            .map(data -> new BulkBan(gateway, data));
+    }
+
```

Comment:
You should also add a method that returns a BulkBanRequestMono, like this:
```Java
public BulkBanRequestMono bulkBan(List<Snowflake> users) {
    return BulkBanRequestMono.of(this, users);
}
```

Reply:
Added in https://github.com/Discord4J/Discord4J/pull/1212/commits/bd0f12180d73edd554dd917ce196a21a67b187e2